src/components/layout/Accordion/AccordionItem.tsx

```diff
@@ -22,6 +23,7 @@ const animationStateClasses = {
 interface PropsBase extends ILocalContainerProps {
 	children: (React.ReactNode & { props?: any[] }) | null;
 	clickArea?: 'icon' | 'all';
+	/** Defaults to "true" for default carot icons. Set to "false" for no icon, set to ReactNode for custom icon */
```

Comment:
Tiny typo, otherwise looks good!

```suggestion
	/** Defaults to "true" for default caret icons. Set to "false" for no icon, set to ReactNode for custom icon. */
```